governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+| Res              | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | Resiliency           | 14.9         |
+| LDF              | [0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50](https://etherscan.io/address/0xC322E8Ec33e9b0a34c7cD185C616087D9842ad50) | Composable           | 11.92        |
+| opensky          | [0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04](https://etherscan.io/address/0x8e67ee3bbeb1743dc63093af493f67c3c23c6f04) | Composable           | 14.9         |
+| David Phelps     | [0xd56e3E325133EFEd6B1687C88571b8a91e517ab0](https://etherscan.io/address/0xd56e3E325133EFEd6B1687C88571b8a91e517ab0) | Composable           | 8.94         |
+| seedlatam.eth    | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d) | Sovereign Finance    | 11.92        |
+| StableLab        | [0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08](https://etherscan.io/address/0xbDE65cf2352ed1Dde959f290E973d0fC5cEDFD08) | Growth               | 14.9         |
+| flipsidegov      | [0x300901243d6CB2E74c10f8aB4cc89a39cC222a29](https://etherscan.io/address/0x300901243d6CB2E74c10f8aB4cc89a39cC222a29) | Growth               | 14.9         |
+
+### Clydesdale Quarterly Surplus Return
+
+If this executive proposal passes, this [MakerDAO Resolution](https://gateway.pinata.cloud/ipfs/QmaGTVioBsCPfNoz9rbW7LU6YuzfgqHDZd92Hny5ACfL3p) will be approved.
+
+This resolution 'requests and ratifies' James Asset PTC Limited to liquidate shares of James Asset Trust (JAT1) and return proceeds to the MakerDAO surplus buffer.
+
+See this forum [post](https://forum.makerdao.com/t/clydesdale-quarterly-return-of-surplus-fund/21291) for additional context.
+
+## Review
+
+Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.
+
+---
+
+## Resources
+
+Additional information about the Governance process can be found in the [Maker Operational Manual](https://manual.makerdao.com).
+
+To add current and upcoming votes to your calendar, please see the [MakerDAO Governance Calendar](https://manual.makerdao.com/makerdao/calendars/governance-calendar).
+
+# Meta-Notes from DAO Masons
+
+- In New Silver, we couldn't really tell what the 'DOC variable' was about or exactly what's being updated. It's in the spreadsheet so it's clearly important, but the originating [post](https://forum.makerdao.com/t/rwa-002-new-silver-restructuring-risk-and-legal-assessment/21417/15) doesn't give much context. Tried to walk the 'clear enough but still vague so it's not technically wrong' line but would have liked to give more context here. Suspect there is some documentation on this that would make it clear, but we didn't find it. We'd be sure to ask about this.
```

Comment:
Yes, docs linked above. They should have been linked in the forum post, but yeah, people forget / can't be bothered.